scripts/apps/authoring-react/authoring-integration-wrapper.tsx

```diff
@@ -323,14 +346,29 @@ export class AuthoringIntegrationWrapper extends React.PureComponent<IPropsWrapp
                     return tab;
                 });
 
+            const {sideWidget} = this.state;
+
             return (
                 <Nav.SideBarTabs
-                    activeTab={this.state.sideWidget?.id}
+                    disabled={sideWidget?.pinned}
+                    activeTab={sideWidget?.id}
                     onActiveTabChange={(val) => {
+                        if (val == null && closedOnRender.closed == true) {
+                            closedOnRender.closed = false;
+                        }
+
+                        const isWidgetPinned = (() => {
+                            if (sideWidget?.id === val && sideWidget.id != null) {
```

Comment:
can `sideWidget` be `null`? You are accessing it twice and once you use optional chaining and second time not.

Reply:
Yes it can be `null` of course. Yes, I am, I had some reason behind that. `val` can also be null, that's why I added the second condition. If `val` is `null` another branch needs to execute.

Reply:
the issue I pointed out to `id` being set to `null` is still not addressed.